invenio_communities/members/services/service.py

```diff
@@ -437,6 +437,10 @@ def _members_search(
             schema=schema,
         )
 
+    def read_memberships(self, identity):
+        """Searches the memberships of a specific user/identity."""
+        return {"memberships": self.config.record_cls.get_memberships(identity)}
```

Comment:
This is not an ideal searialization. Doing it properly would require rework of the actual members cls item.
To be done separately at https://github.com/inveniosoftware/invenio-communities/issues/896